source/overview/license-and-subscription.rst

```diff
@@ -0,0 +1,126 @@
+
+1. Log in to your Mattermost customer profile. If you haven't created a profile, follow the steps provided on the landing page, validate your email address, and then log in.
+2. Choose a subscription type, enter the number of users in the **Order summary** field.
+3. (Optional for E20) You can add Premier Support, the cost of which is automatically added to your order total.
+4. Select **Next Step**.
+5. Enter your billing and payment information.
+6. Accept the **Terms**
+7. Select **Complete**.
+
+Choose **Download the license key**, and navigate to the Mattermost System Console to apply it.
+
+If you experience any problems with your transaction, please contact our Support Team via the form provided in the Customer Portal.
+If possible, keep the error message/number that you received on hand as it may assist with their investigation.
+
+Applying the license key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   If you are not a System Admin, you will not have access to the System Console to apply the license.
+
+Navigate to the **Subscriptions** page of your Mattermost account. Select **Download License** to download the license key for your subscription.
+
+You can apply the license in the Mattermost System Console: **System Console > About > Edition and License**. Follow the steps provided to apply your license key.
+You can also use the `CLI <https://docs.mattermost.com/install/ee-install.html#changing-a-license-key>`__ to apply the license.
+
+Viewing license information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can view your license start date, expiry date, number of users and license type in your account under **Subscription**.
+
+Adding more users to a purchased license
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can add more users during your subscription period without requesting a license.
+
+During the annual renewal, a retroactive charge will be placed for any unique users added during the past subscription period that is
+above the licensed total unique users in the current paid subscription. The retroactive charge per user will be the initial subscription
+cost per user.
+
+Renewing an Enterprise license
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ability to renew your license in the Customer Portal will be available in a future release.
+To renew your license in the meantime, contact your CSM to start the renewal process.
+
+Frequently Asked Questions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Are my credit card details safe?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use `Stripe <https://stripe.com/payments>`_ to process credit card transactions. We do not store any credit card details at any stage. Payments made by credit card are authenticated using `3D Secure <https://support.payfast.co.za/article/96-what-is-3d-secure-visa-secure-mastercard-securecode>`__ which is PCI-DSS compliant.
+
+Should you wish to make payment using another method, please contact our Billing team.
```

Comment:
Absolutely. I'm awaiting feedback on the correct link.

Reply:
@justinegeffen @esadur - I don't think this is the right motion. Let's say a customer wants to use an alternative billing method, do we wait to provision a license until we get the payment? Making a credit card mandatory, solves for a few things: 1) allows us to charge the customer immediately 2) we have a a credit card on hand to charge for renewals. 

If the customer has to wait for billing, do we hold provisioning a license until the customer has paid us or do we give them the license and wait for payment? Given the limitation of our product and we not have the ability to physically limit the product, we loose our leverage once we deliver the license. This means there is an increased risk that customer will not pay us. Further, this will frustrate the customer and billing needs to be involved to indicate when a customer has paid. This would be a manual workload for my team. 

If there is a case to want to pay via alternative method, I like to request that information be pulling from SFDC for paid records. All paid information of a customer sits in SFDC under the financial record which is pushed from Netsuite.

Reply:
@TQuock if the customer is not purchasing with a credit card, they go through the normal Salesforce process with Sales/CSM. They will not get a license key until they either have a successful charge to their credit card or they go through the existing process in SFDC.

Reply:
Great point, thanks @TQuock. 

My understanding around this particular line was that it's related to the other payment methods [listed in the FAQ](https://docs.mattermost.com/overview/faq.html#what-are-the-options-to-purchase-a-subscription-to-mattermost-enterprise-edition). 

@esadur, @wiersgallak - do we want to explicitly mention the different purchase methods in this document? Or should we refer to the link above? Or, should we remove the FAQ content and refer to this doc instead?

Reply:
@rbradleyhaas - Can you expand on this. Does this mean that the customer would not go through the customer portal to register and would go through a sales rep to purchase or renew as part of v1.0? If they are to go through a sales rep, would the customer not have a customer portal? 

I want to understand the motion for this first release. Is it that we only put customers through the customer portal if they are new and want to pay by credit card? Or are we pushing all new customers to the customer portal and giving them the option to pay via credit card. If they elect not to pay via credit card, then we are giving them other options like check/ACH? If the latter is the case, then I am not comfortable with signing off as this is a different motion then what we currently have in place.    

> @TQuock if the customer is not purchasing with a credit card, they go through the normal Salesforce process with Sales/CSM. They will not get a license key until they either have a successful charge to their credit card or they go through the existing process in SFDC.